app/assets/main/react_components/routes/lifestyle_footprints/footprint_form_container/components/footprint_form/FootprintForm/footprint-form-logic.js

```diff
@@ -1,15 +1,15 @@
 const cleanFootprint = (footprint) => {
-  var tempFootprint = { ...footprint };
-  for (var footprintField in tempFootprint) {
+  const cleanedFootprint = {...footprint}
+  for (var footprintField in cleanedFootprint) {
     if (
-      tempFootprint[footprintField] === null ||
-      tempFootprint[footprintField] === undefined
+      cleanedFootprint[footprintField] === null ||
+      cleanedFootprint[footprintField] === undefined
```

Comment:
Slog mig nu att det kanske räcker med att bara kolla på detta vis?

```suggestion
      !cleanedFootprint[footprintField]
```